Add `Relaxed Query Expression Keywords` section to SLB3 release note (#951)
ballerina-platform/ballerina-release: release-notes/swan-lake-beta3-release-note.md

}
``` 

##### Relaxed Query Expression Keywords

Identifiers that are keywords in a query expression context (`where`, `join`, `order`, `by`, `equals`, `ascending`, `descending`, `limit`, `outer`, and `select`) can now be used as ordinary identifiers outside query contexts. i.e., they are no longer required to be quoted identifiers.

> ```ballerina

Reviewer: ```suggestion

```ballerina
```

> // Now allowed.

Reviewer: ```suggestion
// Now allowed. No longer required to use a quoted identifier (`'limit`).
```

int limit = 5;

> // `string:'join()` is no longer required to use quoted identifiers.

Reviewer: ```suggestion
// Now allowed.  No longer required to use a quoted identifier (`string:'join()`).
```

}
``` 

##### Relaxed Query Expression Keywords

> Keywords, that are used in query expression context (`where`, `join`, `order`, `by`, `equals`, `ascending`, `descending`, `limit`, `outer`, `select`) can now be used as identifiers outside the query context.

Reviewer 2: ```suggestion
Keywords that are used in a query expression context (`where`, `join`, `order`, `by`, `equals`, `ascending`, `descending`, `limit`, `outer`, `select`) can now be used as identifiers outside the query context.
```

Reviewer: How about something like

```suggestion
Identifiers that are keywords in a query expression context (`where`, `join`, `order`, `by`, `equals`, `ascending`, `descending`, `limit`, `outer`, and `select`) can now be used as ordinary identifiers outside query contexts. i.e., they are no longer required to be quoted identifiers.
```

```ballerina
// Now allowed.
int limit = 5;

> // `string:'join()` now can be used without initial quote.

Reviewer: ```suggestion
// `string:'join()` is no longer required to use quoted identifiers.
```
